python_modules/libraries/dagster-dg/dagster_dg_tests/cli_tests/test_custom_help_format.py

```diff
@@ -203,7 +203,7 @@ def test_dynamic_subcommand_help_message():
                 │ --help         -h            Show this message and exit.                                                             │
                 ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
                 ╭─ Global options ─────────────────────────────────────────────────────────────────────────────────────────────────────╮
-                │ --cache-dir                                                        PATH  Specify a directory to use for the cache.   │
+                │ --cache-dir                                                        TEXT  Specify a directory to use for the cache.   │
```

Comment:
why

Reply:
The CLI options are sourced both from the CLI and the `tool.dg.cli` section in TOML. It is impossible to load `Path` directly from TOML. It simplifies things to keep the types constant between CLI and TOML (which means using simple types) and perform the normalization in one downstream place.